### DISCUSSION TITLE
-j also saves djbz chunks

### DISCUSSION THREAD

**Discussion by maple7-7-7** [trufanov-nok/minidjvu-mod#22] [Ideas]
Hi Alex,
I just tested your option **-j**, which is designed to save jb2 chunks. Of course, it also saved the related djbz chunks. Thanks. I am wondering if you would like to include in the description of functions the fact that **-j** also produces djbzs. Although a logical expectation, I was not sure that the **-j** option would include the dictionary chunks.

The description could go from
"save pages as jb2 chunks instead of djvu
 implies indirect mode"
to
"save as jb2 chunks and any djbz (dictionary) chunks
 implies indirect mode"

The "save as . . . chunks" expression implies "instead of as DjVu pages."

Thanks.

**Comment by trufanov-nok** [comment]
Hi,
The problem is that indirect mode is djvu+djbz. And this option turns it to jb2+djbz.  Thus I thought that mention of indirect mode should make user expect djbz too. So it's indirect mode with jb2 instead of djvu. In case of second wording: the mention of indirect mode and omitting "instead" may be confusing as user may expect djbz+ both djvu and jb2 then.   

Frankly I added this option for a quite specific case - it allows to skip DjVuLibres' `djvuextract` usage (extracting the sjbz chunk from 1-page djvu) step in "method of separate scans" and will be used in ScanTailor Universal to speed up djvu assembling. I even thought to not mention this option in the help output and thus hide it from the user, but decided that one sooner or later find it and will start asking what it does.   

Btw there is 1 more hidden option in minidjvu-mod - if I recall right `-m` is case sensitive and has different effect. -M is more aggressive and faster as omitting some checks during the character comparision. It's a minidjvu project legacy. I'll certanly delete -M option when finalize matching params and solve current encoding issues (bold detection for ex.).

**Reply by maple7-7-7** [reply]
Thanks for the update.
